helm/helm-app-database/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>helm-app-database</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: Helm: Application + database</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-openshift</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-container-image-docker</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.helm</groupId>
+            <artifactId>quarkus-helm</artifactId>
+            <version>0.0.6-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
+        </dependency>
+        <!-- TODO https://github.com/quarkiverse/quarkus-helm/issues/66 -->
+        <!--        <dependency>-->
+        <!--            <groupId>io.quarkus</groupId>-->
+        <!--            <artifactId>quarkus-hibernate-orm-panache</artifactId>-->
+        <!--        </dependency>-->
+        <!--        <dependency>-->
+        <!--            <groupId>io.quarkus</groupId>-->
+        <!--            <artifactId>quarkus-jdbc-postgresql</artifactId>-->
+        <!--        </dependency>-->
+        <dependency>
+            <groupId>io.quarkus.qe</groupId>
+            <artifactId>quarkus-test-helm</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus.qe</groupId>
+            <artifactId>quarkus-test-openshift</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus.qe</groupId>
+            <artifactId>quarkus-test-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <profiles>
+        <profile>
+            <!-- Disable native build on this module -->
+            <!-- Helm is concerned just about image name, Native compilation is not relevant -->
+            <id>native</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+            </activation>
+            <properties>
+                <!-- To not build the module on Native -->
```

Comment:
This comment is not precise, build is not skipped on native, but fast-jar is created. Stork is using same comment and when I run `mvn clean install` I got
```bash
[INFO] --- quarkus-maven-plugin:999-SNAPSHOT:build (build) @ stork ---
[WARNING] [io.quarkus.stork.deployment.SmallRyeStorkProcessor] The application is using the Stork Kubernetes Service Discovery provider but does not depend on the `quarkus-kubernetes-client` extension. It is highly recommended to use the `io.quarkus:quarkus-kubernetes-client` extension with the Kubernetes service discovery. 
To add this extension:
 - with the quarkus CLI, run: `quarkus ext add io.quarkus:quarkus-kubernetes-client`
 - with Apache Maven, run: `./mvnw quarkus:add-extension -Dextensions="io.quarkus:quarkus-kubernetes-client"`
 - or just add the `io.quarkus:quarkus-kubernetes-client` dependency to the project
[WARNING] [io.quarkus.vertx.web.deployment.ReactiveRoutesProcessor] Route io.quarkus.ts.stork.PongResource#pong() can match the same request and has the same order [0] as:
	- io.quarkus.ts.stork.PongReplicaResource#pong()
[WARNING] [io.quarkus.vertx.web.deployment.ReactiveRoutesProcessor] You can use @Route#order() to ensure the routes are not executed in random order
[INFO] [io.quarkus.deployment.QuarkusAugmentor] Quarkus augmentation completed in 1277ms
```

If your intention is truly skip the build, I'd suggest to set also
```
<quarkus.build.skip>true</quarkus.build.skip>
```
which leads to
```
[INFO] --- quarkus-maven-plugin:999-SNAPSHOT:build (build) @ stork ---
[INFO] Skipping Quarkus build
```
Alternatively, you can adjust the comment  (as well as `<!-- Disable native build on this module -->` one) so I can clearly understand what's your intention.

Thank you

Reply:
Thanks @michalvavrik I wan't aware about this property `<quarkus.build.skip>true</quarkus.build.skip>` :heart: